packages/desktop/components/popups/CreateDelegationPopup.svelte

```diff
@@ -50,12 +50,7 @@
         !addressError &&
         transactionInfo.preparedTransaction &&
         !transactionInfo.preparedTransactionError
-    $: displayManaBox =
-        !!accountAddress &&
-        !addressError &&
-        validAmount &&
-        transactionInfo.preparedTransaction &&
-        !transactionInfo.preparedTransactionError
+    $: displayManaBox = !!accountAddress && !addressError && validAmount
```

Comment:
How is displaying the `ManaBox` or not, related to the confirm button? Shouldn't you be tweaking `confirmAllowed` ?